test/tests/functional/pbs_qrun.py

```diff
@@ -100,30 +100,46 @@ def test_qrun_hangs(self):
         """
         node = self.mom.shortname
         time1 = int(time.time())
+        walltime = 1
+        self.server.manager(MGR_CMD_SET, SCHED,
+                            {'scheduling': 'False'})
+        err_msg = 'qrun: Not enough free nodes available'
         while True:
             self.server.manager(MGR_CMD_SET, NODE,
                                 {'resources_available.ncpus': 1}, id=node)
```

Comment:
Can you move this outside of the while?  It doesn't change each iteration

Reply:
@bhroam  that is already outside of the while.